docker-images/kafka/cruise-control-scripts/cruise_control_tls_prepare_certificates.sh

```diff
@@ -24,13 +24,16 @@ function create_keystore {
 
 echo "Preparing certificates for internal communication"
 STORE=/tmp/cruise-control/replication.truststore.p12
+rm -f "$STORE"
```

Comment:
out of curiosity, why this remove?

Reply:
When the container restarts (not when the Pod is deleted but just when the container restarts), the volume stays. But because the password is generated at the startup scripts it will not match anymore and the process will fail. I think that was to some extent issue even before this PR. Making sure the store does not exist avoids these issues, since you can always use the new password from scratch.

Reply:
Makes sense. Yeah my question was mostly related to this PR as you said because the problem could have been even before.

Reply:
TBH, I know I saw the problem before on occasions. But it is not clear how it could have happened. With this PR the issue is clear -> the volume where these are now created is bound to the pod lifecycle so they should keep the files normally. But before this PR it IMHO we were creating them in the container context so I'm not sure why they sometimes seemed to survive. 🤷‍♂️